extensions/silero_tts/tts_preprocessor.py

```diff
@@ -2,6 +2,7 @@
 
 from num2words import num2words
 
+#punctuation = r'[\s,.?!/)"\'\]>]“”'
```

Comment:
It needs to be inside the right-most `]`

Reply:
That's a regex `[ ]` meaning any one of the characters inside the brackets

Reply:
I'm guessing you don't use it. Pycharm Community is free, and it has a regex checker built into the editor, along with many other tools, like fixing the whitespace issue from before.

Reply:
Eh, I hadn't noticed it was a regex.

Reply:
https://www.jetbrains.com/webstorm/guide/tips/check-regex/

Reply:
> Eh, I hadn't noticed it was a regex.

Fair

Reply:
I just checked while I was fixing some other things I found, and those fancy quotes were already replaced with empty strings. I changed and added some more fancy quotes to a filter with the unicode literals, which makes them easier to tell apart right next to each other.
The changes you had are in there and cleaned up, and I think it's good again.